### ISSUE TITLE
Is TD-DFT supported on v1.2.1 ?

### ISSUE THREAD

**Issue by black-suit** [psi4/psi4#2071]
![スクリーンショット 2020-12-20 16 13 24](https://user-images.githubusercontent.com/59173753/102707472-9252b980-42de-11eb-8048-896f3ddd1095.jpg)
(↑**tdscf_excitations can't not import**)

![スクリーンショット 2020-12-20 16 14 56](https://user-images.githubusercontent.com/59173753/102707475-9d0d4e80-42de-11eb-96ad-5b60478590d2.jpg)
(↑**I can't use TDSCF-STATES**)

**I use psi4 v1.2.1 in psi4conda. To execute TD-DFT calculations, what should I do ?**

**Comment by JonathonMisiewicz** [comment]
Upgrade your version of Psi4. If you want TD-DFT, I suggest you use the [nightly build here](https://psicode.org/installs/v132/).

TD-DFT calculations are being added in version 1.4. You can see a guide on how to perform TD-DFT [here](https://www.youtube.com/watch?v=MmG-g3SEFbk).

**Comment by black-suit** [comment]
Thank you. I'll try it.

**Comment by black-suit** [comment]
Thank you. I'll try it.

**Comment by black-suit** [comment]
I have a question, Psi4 in psi4conda can't update?

**Comment by loriab** [comment]
Somewhere I saw a question about whether psi4 in psi4conda could update. 

tl;dr no, create a new environment instead, as you'll get the latest, and it's far more predictable.

Technically, yes, `conda update psi4 -c psi4` should get you the latest stable release, v1.3.2 from mid-2019, or `conda update psi4 -c psi4/label/dev` should get you the latest nightly build from this week. However, practical advice is that whenever your psi4 is more than a few months old, don't update, just create a new environment, `conda create -n <env-name-of-your-choice> python=3.<python-version-of-your-choice> psi4 -c psi4/label/dev`, activate that env, and use that new psi4 instead of the (remaining, still functional) psi4 in the main environment of psi4conda. The reasons that a freshly solved environment is better is that: (1) Far upstream packages like mkl or numpy or hdf5 update over time and new psi4 may not be built against the older ones in the psi4conda env, so you don't get the newest psi. (2) To make psi4-served conda packages easy to use for both compiling and installing binary psi4, I don't pin dependencies very tightly, so the constraints of the env may not allow all the latest packages to be pulled or even a consistent set of packages. Same advice for any psi4 install, not just psi4conda.
